.github/workflows/master.yml

```diff
@@ -14,9 +14,19 @@ jobs:
           run: printf "$GITHUB_SHA" > .application-version
```

Comment:
Since we have a Concourse job that does this already, pushing to GCR on master isn't needed .

Reply:
Removed from master workflow, thanks!